src/main/java/frc/team4533/robot/Robot.java

```diff
@@ -1,38 +1,43 @@
 package frc.team4533.robot;
 
-import frc.team4533.robot.subsystems.DriveSystem;
-
 import edu.wpi.first.wpilibj.IterativeRobot;
 import edu.wpi.first.wpilibj.command.Scheduler;
-import edu.wpi.first.wpilibj.livewindow.LiveWindow;
+import frc.team4533.robot.subsystems.DriveSystem;
 
 public class Robot extends IterativeRobot {
 
-
-	//public static NameOfAutonomousCommand autonomousCommand;
+	// public static NameOfAutonomousCommand autonomousCommand;
 	public void robotInit() {
 		DriveSystem.initialize();
 	}
+
 	public void disabledInit() {
 
 	}
+
 	public void disabledPeriodic() {
 		Scheduler.getInstance().run();
 	}
+
 	public void autonomousInit() {
 
 	}
+
 	public void autonomousPeriodic() {
 		Scheduler.getInstance().run();
 	}
+
 	public void teleopInit() {
-//		if(autonomousCommand != null) {
-//			autonomousCommand.cancel();
-//		}
+		// if(autonomousCommand != null) {
+		// autonomousCommand.cancel();
+		// }
+		DriveSystem.getInstance().setPIDFValues(0.113333, 0, 0, .1097);
```

Comment:
Moving the PIDF settings into a method is a great idea. Nice work!

Reply:
We discussed removing driveAction() on Saturday and agreed that it should go away as it would no longer be necessary.